models/User.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace models;
+
+class User extends BasicModel{
+
+	/**
```

Comment:
Fix indentation. Probably a generic setting for entire project. It seems that all files are affected.